includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -2951,6 +2962,48 @@ public static function get_error_title_from_code( $validation_error ) {
 					$title .= sprintf( ': <code>%s</code>', esc_html( $validation_error['property_name'] ) );
 				}
 				return $title;
+			case AMP_Tag_And_Attribute_Sanitizer::DUPLICATE_ONEOF_ATTRS:
+				$title = __( 'Mutually exclusive attributes encountered', 'amp' );
+				if ( ! empty( $validation_error['duplicate_oneof_attrs'] ) ) {
+					$title .= ': ';
+					$title .= implode(
+						', ',
+						array_map(
+							static function ( $attribute_name ) {
+								return sprintf( '<code>%s</code>', $attribute_name );
+							},
+							$validation_error['duplicate_oneof_attrs']
+						)
+					);
+					return $title;
+				}
+				break;
+			case AMP_Tag_And_Attribute_Sanitizer::MANDATORY_ONEOF_ATTR_MISSING:
+			case AMP_Tag_And_Attribute_Sanitizer::MANDATORY_ANYOF_ATTR_MISSING:
+				$attributes_key = null;
+				if ( AMP_Tag_And_Attribute_Sanitizer::MANDATORY_ONEOF_ATTR_MISSING === $validation_error['code'] ) {
+					$title          = __( 'Missing exclusive mandatory attribute', 'amp' );
+					$attributes_key = 'mandatory_oneof_attrs';
+				} else {
+					$title          = __( 'Missing at least one mandatory attribute', 'amp' );
```

Comment:
This looks good, it's much more descriptive than before:

![amp-list-here](https://user-images.githubusercontent.com/4063887/74594145-a8c22180-4ff8-11ea-8bee-20a41dec59e4.png)